src/styles/card/card.scss

```diff
@@ -0,0 +1,113 @@
+.main {
+  padding: 50px 40px;
+  height: 862px;
+  max-width: 1200px;
+
+  /* Центрування елемента горизонтально */
+  margin: 0 auto;
+
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  grid-template-rows: repeat(2, 408px);
+  gap: 46px 48px;
+
+  justify-content: center;
+  align-items: center;
+}
+
+@media (min-width: 488px) {
+  .main {
+    grid-template-columns: repeat(2, 200px);
+  }
+}
+
+@media (min-width: 768px) {
+  .main {
+    grid-template-columns: repeat(3, 200px);
+  }
+}
+
+@media (min-width: 1024px) {
+  .main {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  // width: 200px;
+  border: 1px solid white;
+  box-sizing: border-box;
+  background-color: #fff;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  transition: transform 300ms;
+
+  &__title {
+    margin: 40px 0 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+  }
```

Comment:
According to the code style guide, the margin and padding properties should be grouped together and placed above the other properties.